modules/plugin/pom.xml

```diff
@@ -59,14 +62,10 @@
     <module>imagemosaic-jdbc</module>
     <module>jdbc</module>
     <module>jp2k</module>
+    <module>mongodb</module>
     <module>property</module>
     <module>shapefile</module>
     <module>svg</module>
-    <module>coverage-multidim</module>
-    <module>geopkg</module>
```

Comment:
So this change kicked geopkg out of the build? Ideally a module promotion should just do that, promote the module, all the other improvements should be in a different commit (if not different PR), because they are not related, and are easy to get lost in the movement of so many files (github reports 200+ files changed in this PR). Thankfully I spotted this one at the source...

Reply:
Thanks for the catch Andrea, I appreciate it.
I initially tried moving this to plugin, but it did not fit (since it defines new API).